lib/operations/operations_api.rb

```diff
@@ -0,0 +1,144 @@
+require 'hawkular'
+require 'websocket-client-simple'
+require 'json'
+
+class Proc
```

Comment:
Are you sure that this should go into the root namespace and not under Hawkular::Operations ?